manifests/l/Libretro/RetroArch/Nightly/1.10.3/Libretro.RetroArch.Nightly.installer.yaml

```
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-22598-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Libretro.RetroArch.Nightly
+PackageVersion: 1.10.3
```

```suggestion
PackageVersion: 1.10.3.0
```
This must match Control Panel.